func/mpi/migrate.cpp

```diff
@@ -62,6 +63,14 @@ void doBenchmark(int nLoops)
     MPI_Comm_rank(MPI_COMM_WORLD, &rank);
     MPI_Comm_size(MPI_COMM_WORLD, &worldSize);
 
+    // Measure time spent migrating
+    // timespec timeStart{}, timeEnd{};
```

Comment:
Does this line need to be here commented out? Can we delete?

Reply:
I forgot to remove this one :/

Thanks